### ISSUE TITLE
Nothing Updating

### ISSUE THREAD

**Issue by TannerLay** [UnamSanctam/UnamWebPanel#52]
Hey I tested the miner on myself to see if my webpanel would update and it hasn't. About 20 minutes ago I got the same ajax error that other users reported, but you said it was fixed so I am not sure. Only fix I have tried is replacing the db with the link you provided to the other user with the same issue.

Everything should have been done right, this is the API link I used: http://zap543185-1.plesk10.zap-webspace.com/api/endpoint.php

![image](https://user-images.githubusercontent.com/73258424/148865264-4bfa2096-3a3b-49b2-81c1-7f27e11e5b65.png)

**Comment by UnamSanctam** [comment]
Do you mean that you don't get any miners or do you mean that the miners sometimes corrupt the db?

**Comment by TannerLay** [comment]
I have tested the miner on myself and its not popping up on the webpanel. I also checked permissions for the db and it has rights to everything.

**Comment by UnamSanctam** [comment]
It looks like your plesk is auto-redirecting it to https (for me at least) when you don't have a valid certificate.

**Comment by TannerLay** [comment]
You're right. This stupid shit isn't letting me change it though.

![image](https://user-images.githubusercontent.com/73258424/148870089-7c61d817-b8f7-4012-85f3-30862f352d08.png)

**Comment by TannerLay** [comment]
Since it redirects even tho I don't have a valid ssl cert. Can't I just change the API site link to https and it should work?

**Comment by UnamSanctam** [comment]
Problem is that you don't have a valid certificate so it cannot connect.

**Comment by TannerLay** [comment]
Alright, I fixed that issue. Still not showing up unfortunately even after uninstalling miner and retrying it on my system.

**Comment by UnamSanctam** [comment]
Run the checker and make sure that the endpoint URL is correct.

**Comment by TannerLay** [comment]
This is all the checker is showing even though I ran the miner?

![image](https://user-images.githubusercontent.com/73258424/148877997-62c9d6b3-e2d9-42a9-9bb2-a067b9e8b47c.png)

**Comment by UnamSanctam** [comment]
Then the miner isn't running, check that all of your settings are correct. If you want me to check through them then you can go to the "Main" tab and then press "Save", then zip that file and post it here.

**Comment by TannerLay** [comment]
I tried checking my settings again, still having the issue. Here are the settings I am using:
[New WinRAR ZIP archive.zip](https://github.com/UnamSanctam/UnamWebPanel/files/7850375/New.WinRAR.ZIP.archive.zip)

The API link I am using in the miner is: http://zap543185-1.plesk10.zap-webspace.com//api/endpoint.php

**Comment by UnamSanctam** [comment]
It works for me so make sure that your GPU drivers are new enough.

**Comment by TannerLay** [comment]
Can I bind the miner with mercurial grabber without issues? If so do I just use your Unam downloader?

**Comment by UnamSanctam** [comment]
Yes that should work fine, and binder (or downloader) should work.

**Comment by TannerLay** [comment]
So I had a different pc run it (3070 graphics card), and I'm not getting any response under the checker or the web panel.

**Comment by TannerLay** [comment]
For some reason I just genuinely don't think the miner/panel is working for any pc with these settings for some odd reason.

**Comment by UnamSanctam** [comment]
Try disabling "Startup" to see if it's an installation issue or something else.

**Comment by TannerLay** [comment]
That fixed the issue on both the checker and webpanel lol. Won't that be an issue though? So now it won't startup when pc resets?

**Comment by UnamSanctam** [comment]
Yes but that means the issue is with installing it, try enabling "Startup" but change the "Filename" to something else.

**Comment by TannerLay** [comment]
That seemed to have worked thanks!! One last thing, do you recommend a certain pool for XMR? I know it can be iffy with passwords and security and shit. I am trying nanopool but idk if its working.

**Comment by UnamSanctam** [comment]
I know a lot of people use MoneroOcean and 2miners as well, they are both generally good. Nanopool is good because of the fact that they don't ban but they do have a high share difficulty and such which are downsides. You can also mine Raptoreum with the newest miner version which currently should be more profitable than Monero but you can also just change that from within the web panel.

**Comment by Yara-web** [comment]
Guys I have the same problem in the web panel miners are not shown

**Comment by UnamSanctam** [comment]
>Guys I have the same problem in the web panel miners are not shown

You should open your own issue but you can see in this whole issue how it was solved here.
